src/presentation/styles/styledComponents/StyledRect.js

```diff
@@ -1,17 +1,28 @@
 import styled from 'styled-components';
-import {chosenPrompt, activeTab} from './StyledHeader.js'
+import {chosenPrompt, activeTab, types} from './StyledHeader.js'
 import '../styleSheets/sass/StyledHeader.scss'
 import React from 'react';
 import { OutterRect, InnerRect, outterTextStyle, innerTextStyle } from './StyledConstants.js';
 
+var CurrentMessage = {[types[0]]: " - Hello, here are some things I can help you with:",[types[1]]: "- Hi, here are some things I can help you with:"}
+
+
+
+
 // display outter rectangle then inner rectangle inside it
 export const StyledRect = () => (
     <OutterRect className="container" style={outterTextStyle}>
         <div style={{fontWeight: 'bold'}}>
-            {activeTab}
```

Comment:
Good modification of my code, try to use more comments now especially because there are multiple messages and inner rects that do different things now.